schemas/expconf/v0/resources.json

```diff
@@ -52,6 +52,13 @@
             "maximum": 99,
             "default": null
         },
+        "resource_manager": {
+            "type": [
+                "string",
+                "null"
+            ],
+            "default": ""
+        },
```

Comment:
maybe this is a dumb question... but why do users need to specify the resource manager and the resource pool?

Why don't we just use resource pools and they specify those?  This is a layer of hierarchy that has never made sense to me.

Reply:
Ah this is for use in the multi-RM project -- multiple resource managers can have resource pools with the same name. In that case, we need to use the resource manager name to qualify & identify it.

Reply:
why don't we just disallow multiple resource pools with the same name?

Reply:
I hear the answer to this is "because slurm".

Gross, but ok.

Reply:
FWIW @rb-determined-ai , the way we're approaching this is users will only need to specify if there is a conflict, still. Otherwise no one knows this param exists.